docs/0-user-guides/1-etlas-user-guide/3-configuration/1-dependency-management.md

```diff
@@ -108,13 +108,11 @@ The `maven-repos` field can take a URL to a Maven repository OR a keyword from t
 | Keyword                    | Maven Repository URL                                     |
 | -------------------------- | -------------------------------------------------------- |
 | `central`                  | `https://repo1.maven.org/maven2/`                        |
-| `javaNet1`                 | `http://download.java.net/maven/1/`                      |
+| `javaNet1`                 | `https://maven.java.net/content/groups/public/`          |
 | `sonatype:[type]`          | `https://oss.sonatype.org/content/repositories/[type]`   |
 | `jcenter`                  | `https://jcenter.bintray.com/`                           |
-| `bintray:[owner]:[repo]`   | `https://dl.bintray.com/[owner]/[repo]/`                 |
```

Comment:
bintray is gone. They wanted to kill jcenter as well but too many people protested, so it has an indefinite reprieve.